### DISCUSSION TITLE
BlueRetro on Nintendo Ds works

### DISCUSSION THREAD

**Discussion by davidxgames** [darthcloud/BlueRetro#272] [General]
I used Blue Retro Parallel 1P (12 buttons) adapter cable (NeoGeo, Supergun, JAMMA) the .bin parallel 1p_od_spiffs and the schematic 1P_OD, it works perfect, I made an NDS CONSOLIZER and it works very well, it can also be used in gb gbc and gba, or any console that has 12 buttons or less. 

I leave you a youtube video, it is in Spanish since I do not speak English. https://youtu.be/5zOBSAA-P20 It's very simple, but it's what I can do with what I know.

schematic https://twitter.com/XGAMES_VJ/status/1523047713004146689

**Comment by darthcloud** [comment]
No need for level shifter with system using 3.3v and less.

**Reply by Demonic098** [reply]
What firmware you are using on NDS lite (stock or modified FW for component video output) on this project ?

> I used Blue Retro Parallel 1P (12 buttons) adapter cable (NeoGeo, Supergun, JAMMA) the .bin parallel 1p_od_spiffs and the schematic 1P_OD, it works perfect, I made an NDS CONSOLIZER and it works very well, it can also be used in gb gbc and gba, or any console that has 12 buttons or less.
> 
> I leave you a youtube video, it is in Spanish since I do not speak English. https://youtu.be/5zOBSAA-P20 It's very simple, but it's what I can do with what I know.
> 
> schematic https://twitter.com/XGAMES_VJ/status/1523047713004146689

**Reply by davidxgames** [reply]
https://github.com/LostNintendoHistory/Lost-NDS-TV

**Reply by Demonic098** [reply]
thx
